.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -1,5 +1,6 @@
 name: APIServer compatibility
 on:
+  workflow_dispatch:
```

Comment:
The [workflow_dispatch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) can trigger the workflow manually. Does that mean all developers can trigger the workflow at anytime?

Reply:
> Do you have any concerns about this PR? If you don't want to be able to manually call these two workflows currently, then I think at least APIServer compatibility should not fail quickly,

Yes, it makes sense to me that let the scheduled workflow continue to run and avoid canceling due to other job failures.
But manually triggering the workflow is concerning.

Reply:
kindly ping @liangyuanpeng

Reply:
Hi @RainbowMango  I have move the change of `fail-fast: false` to https://github.com/karmada-io/karmada/pull/4238 and there are fix a problem of timeout. 

I want to keep this PR is open and i think let the workflow support workflow dispatch still relevant. Because I have done some exploration on combining this scheduled task with prow, I hope I will have an update on this aspect soon.